src/main/java/seedu/lifeasier/commands/DeleteNotesCommand.java

```diff
@@ -20,14 +20,15 @@
     private static Logger logger = Logger.getLogger(DeleteNotesCommand.class.getName());
     private String title;
     private boolean isEmpty = true;
+    private int arraySize = 10000;
 
     public DeleteNotesCommand(String title) {
         this.title = title;
     }
 
     private void findTitle(Ui ui, NoteList notes, String title, NoteHistory noteHistory) throws TitleNotFoundException {
         logger.log(Level.INFO, "Start for finding title in note list");
-
+        int[] matchArr = new int [arraySize];
```

Comment:
You might want to consider a new name for the array. Maybe something like "noteMatches"?